lib/surface/components/form/color_input.ex

```diff
@@ -11,7 +11,7 @@ defmodule Surface.Components.Form.ColorInput do
   ## Examples
 
   ```
-  <ColorInput form="user" field="name" opts={{ [autofocus: "autofocus"] }}>
+  <ColorInput form="user" field="color" opts={{ [autofocus: "autofocus"] }}>
```

Comment:
You can remove the brackets in all examples if you want :)

```
<ColorInput form="user" field="color" opts={{ autofocus: "autofocus" }}>
```

Reply:
Indeed :+1: